tests/opcache/basic.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+test caching that calls are not traced works with opcache's preload script
+--INI--
+opcache.enable_cli=1
+--FILE--
+<?php
+
+if (!extension_loaded('Zend OPcache')) die("opcache is required for this test\n");
```

Comment:
I'm guessing this isn't in a `SKIPIF` to make the test fail when OPcache isn't loaded? And that's why these tests are outside of the **tests/ext** directory? And we don't ship them with PECL for all of these reasons as well? :)

Reply:
Correct.